Add content about caggs on multi-node, plus pointers (#611)
timescale/docs: timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md

transaction is very quick, and most of the work happens during the second
transaction, to ensure that the work does not interfere with other operations.

## Using continuous aggregates in a multi-node environment
If you are using Timescale in a multi-node environment, there are some
additional considerations for continuous aggregates.

> When you create a continuous aggregate within a multi-node environment, you must
> create the continuous aggregate on the access node. While it is possible to
> create a continuous aggregate on data nodes, it does not work as expected, and
> could cause problems within your multi-node environment.

Reviewer: Suggest a slightly stronger wording regarding not creating continuous aggregates on the data node.
```suggestion
When you create a continuous aggregate within a multi-node environment, the continuous aggregate should be created on the access node. While it is possible to
create a continuous aggregate on data nodes, it will interfere with the continuous aggregates on the access node and will cause problems with the continuous aggregates on the access node.
```

> When you refresh a continuous aggregate on an access node, it computes a single
> window to update the time buckets. This could slow down your query, especially
> if your data nodes are remote.

Reviewer: Added some clarifications. The connection between a single window and why this would make it slow is not clear.

```suggestion
When you refresh a continuous aggregate on an access node, it computes a single
window to update the time buckets. This could slow down your query if the actual number of rows that were updated is small, but widely spread apart. This is aggravated if the network latency is high if, for example, you have remote data nodes.
```

Invalidation logs are on kept on the data nodes, which is designed to limit the
amount of data that needs to be transferred. However, some statements send
invalidations directly to the log. For example, they might drop a chunk or
truncate a hypertable. This action could slow down performance, in comparison to

> a local update. Additionally, ff don't need to do a lot of refreshes, the

Reviewer: "ff"? Not sure what it was supposed to be.

Author: ```suggestion
a local update. Additionally, if you don't need to do a lot of refreshes, the
```

transaction is very quick, and most of the work happens during the second
transaction, to ensure that the work does not interfere with other operations.

## Using continuous aggregates in a multi-node environment
If you are using Timescale in a multi-node environment, there are some
additional considerations for continuous aggregates.

When you create a continuous aggregate within a multi-node environment, the
continuous aggregate should be created on the access node. While it is possible
to create a continuous aggregate on data nodes, it will interfere with the

> continuous aggregates on the access node and will cause problems with the
> continuous aggregates on the access node.

Reviewer: Some repetition, sorry about that. Not sure if we should avoid the repetition earlier as well.
```suggestion
continuous aggregates on the access node and will cause problems on the access node.
```

Author: Ugh, just noticed the 'wills' as well.

> continuous aggregate should be created on the access node. While it is possible
> to create a continuous aggregate on data nodes, it will interfere with the
> continuous aggregates on the access node and will cause problems with the
> continuous aggregates on the access node.

Author: ```suggestion
continuous aggregate should be created on the access node. While it is possible
to create a continuous aggregate on data nodes, it interferes with the
continuous aggregates on the access node and can cause problems.
```

When you refresh a continuous aggregate on an access node, it computes a single
window to update the time buckets. This could slow down your query if the actual
number of rows that were updated is small, but widely spread apart. This is
aggravated if the network latency is high if, for example, you have remote data
nodes.

Invalidation logs are on kept on the data nodes, which is designed to limit the

> amount of data that needs to be transferred. However, some statements send
> invalidations directly to the log. For example, they might drop a chunk or
> truncate a hypertable. This action could slow down performance, in comparison to

Reviewer: ```suggestion
amount of data that needs to be transferred. However, some statements send
invalidations directly to the log, for example, when dropping a chunk or
truncate a hypertable. This action could slow down performance, in comparison to
```

> a local update. Additionally, if you don't need to do a lot of refreshes, the
> invalidation logs could get very large, and it could cause performance issues.

Reviewer: ```suggestion
a local update. Additionally, if you have infrequent refreshes but a lot of changes to the hypertable, the
invalidation logs could get very large, which could cause performance issues.
```

> Make sure you are maintaining your invalidation log size to avoid this.

Reviewer: ```suggestion
Make sure you are maintaining your invalidation log size to avoid this, for example, by refreshing the continuous aggregate frequently.
```
